libkbfs/tlf_journal_test.go

```diff
@@ -548,6 +549,20 @@ func requireJournalEntryCounts(t *testing.T, j *tlfJournal,
 	require.Equal(t, expectedMDEntryCount, mdEntryCount)
 }
 
+func testMDJournalGCd(t *testing.T, j *mdJournal) {
```

Comment:
That's how I started doing it, but none of those tests fully remove all flushed entries at the end of the test and I didn't think it was important enough to add new tests for this.